plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -348,6 +351,13 @@ export class PersonState {
      * @returns true if the properties were changed, false if they were not
      */
     private applyEventPropertyUpdates(personProperties: Properties): boolean {
+        // this relies on making changes to the object instance, so...
+        // if we should not update the person,
+        // we return early before changing any values
+        if (NO_PERSON_UPDATE_EVENTS.has(this.event.event)) {
+            return false
+        }
```

Comment:
The problem with moving this here is that the browser etc on the event could now be wrong, but folks shouldn't be using that anyway, so let's ship it and update when we re-work the pipeline